internal/scalibrextract/vcs/gitrepo/extractor.go

```diff
@@ -84,23 +86,26 @@ func (e Extractor) FileRequired(fapi filesystem.FileAPI) bool {
 func (e Extractor) Extract(_ context.Context, input *filesystem.ScanInput) ([]*extractor.Inventory, error) {
 	// Assume this is fully on a real filesystem
 	// TODO: Make this support virtual filesystems
```

Comment:
nit: add a comment to explain why we call filepath.Dir?

Reply:
Done